pegasus/sites.v3/code.org/public/css/dance-landing/dance.css

```diff
@@ -0,0 +1,14 @@
+.dance-party-h2 {
+  font-size: 20px;
+}
+
+.dance-img-placeholder {
+  border-radius: 5px;
+  background-color: grey;
+  opacity: 0.2;
+}
+
+.dance-chevron-icon {
+  font-family: FontAwesome;
```

Comment:
You shouldn't need this styling.  We have a font-awesome.css file that handles this for you: https://github.com/code-dot-org/code-dot-org/blob/47379314467b1bad950908600ef9473b893dd8e7/pegasus/sites.v3/code.org/public/css/font-awesome.css#L428 Just use the class for your `%i` element. Here's an example of how it's done with a different icon: https://github.com/code-dot-org/code-dot-org/blob/e608d7e1170c51c0689ca2ecaef5b9dc0576a647/shared/haml/user_header.haml#L32